src/testdir/test_writefile.vim

```diff
@@ -257,6 +269,15 @@ func Test_write_errors()
   let long_fname = repeat('n', 5000)
   call assert_fails('exe "w " .. long_fname', 'E75:')
   call assert_fails('call writefile([], long_fname)', 'E482:')
+
+  " Test for writing to a block device on Unix-like systems
+  if has('unix') && getfperm('/dev/loop0') != ''
```

Comment:
Maybe safer if we also check `getftype('/dev/loop0') ==  'bdev'`?

I also wonder what happens if we run this test as root.

Reply:
I ran the test as root and the test passed.

I see some other test failure in FreeBSD (Cirrus CI). I am not able to reproduce this failure in my local FreeBSD system.